templates/patterns/gallery/pattern-gallery.html.twig

```diff
@@ -0,0 +1,29 @@
+{#
+/**
+ * @file
+ * Gallery pattern.
+ */
+#}
+{% extends '@oe-bcl/gallery' %}
+
+{% set toggle_collapsed = 'view the full gallery (%d)'|t %}
+{% set toggle_expanded = 'view less'|t %}
+{% set title_tag = title_tag|default('h2') %}
+{% set items = bcl_gallery_items(items) %}
```

Comment:
think its best if we account also for empty items

Reply:
That is done in the bcl template, isn't that enough?

Reply:
no cuz bcl_gallery_items complains, we agreed on calling |default here

Reply:
Done